unabridged-works-volume-4/original/02-life-and-immortality.adoc

```diff
@@ -1721,33 +1859,39 @@ This testimony Paul gives, Eph. 5:13-14.
 "`Whatsoever doth make manifest is light.`" Wherefore he saith,
 "`Awake thou that sleepest,`" etc.
 
-Thirdly, It hath a quickening property.
+[.numbered]
+_Thirdly,_ It hath a quickening property.
 For it is the light of life, the light of Christ`'s Spirit, which is living and powerful;
 and it hath life and power in it, and quickeneth and raiseth the man to whom it reacheth,
 and giveth him ability to arise from the dead.
 
-Fourthly, It hath a cleansing, a sanctifying, a purifying, property.
+[.numbered]
+_Fourthly,_ It hath a cleansing, a sanctifying, a purifying, property.
 Oh! how clean it makes within!
 The light is pure, and it maketh pure.
 All the defilement and corruption is in the darkness; none of it is in the light,
 but the light purgeth it away, where it findeth entertainment.
 
-Fifthly, It hath a uniting and separating property.
+[.numbered]
+_Fifthly,_ It hath a uniting and separating property.
 It unites to God; it unites to that which is pure; and it separates from sin, Satan,
 darkness, and whatever is impure.
 
-Sixthly, It hath a preserving nature and property.
+[.numbered]
+_Sixthly,_ It hath a preserving nature and property.
 It preserves the man in whom it dwells; and where it hath power, and rules,
 it keeps out the darkness; so that darkness cannot break in where it hath place,
 and is kept to: nor can the mind break out from the Lord, which keeps in the sense,
 savor, and seasoning of the light of his Spirit.
 
+====
+
 Indeed time would fail me, to speak of the excellent nature, virtues,
 and properties thereof; but come, taste, and see how sweet the light is,
 and what a pleasant thing it is to behold the shinings of this Sun.
 
 [.discourse-part]
-Question. 10. Doth God visit all men with this light?
+_Question. 10._ Doth God visit all men with this light?
```

Comment:
Ack, I botched these Questions, 10 - 15.  Can you remove the first period in each, so that they read like `_Question 10._` instead, for all 5 of them? Thanks!